example_plugin.py

```diff
@@ -42,10 +60,25 @@ class InputParams:
     """
     name: typing.Annotated[
         typing.Union[
-            typing.Annotated[FullName, annotations.discriminator_value("fullname")],
-            typing.Annotated[Nickname, annotations.discriminator_value("nickname")],
+            typing.Annotated[FullName, annotations.discriminator_value("fullname"), schema.name("Full name")],
```

Comment:
I would have considered applying the same formatting style as in line 41-47 here as well.